avm/ptn/ai-platform/baseline/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -11,8 +11,9 @@ metadata description = 'This instance deploys the module in alignment with the b
 @maxLength(90)
 param resourceGroupName string = 'dep-${namePrefix}-aiplatform-baseline-${serviceShort}-rg'
 
-@description('Optional. The location to deploy resources to.')
-param resourceLocation string = deployment().location
+// Enforce uksouth to avoid restrictions around VM zones in certain regions
```

Comment:
Enforcing the location, as there are no restrictions regarding availability zones for the VM in UK South

![image](https://github.com/user-attachments/assets/cf8af91a-23a4-4367-8ed1-76ea11b13ef7)